StudCity.Frontend/src/components/Sidenav/FullNavbar.jsx

```diff
@@ -1,14 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Switch from '../Switch';
 import MenuComponents from './MenuItems';
+import Button from '../../UI/Button';
+import CustomDialog from '../../UI/CustomDialog'
+import CreateChatModal from './CreateChatModal';
 
 const FullNavbar = ({ active, activeName }) => {
+	const [isOpenCreateChatModal, setIsOpenCreateChatModal] = useState(false);
+
+	const handleClick = () => {
+		setIsOpenCreateChatModal(true);
+	}
+
+	const handleClose = (isClose) => {
```

Comment:
same thing as I mentioned here:

> from the name of the function I assume that it would only set state showDialogForAvatar to false, if that's the case I think you would better wright function as follows:
> const handleCloseAvatarDialog = () => setShowDialogForAvatar(false);

do it in the same way as handleClick

Reply:
I think it is not necessary to change this logic because it is used in many components

Reply:
you just need to change this particular function